internal/faustjs.org/docs/guides/custom-post-types.mdx

```diff
@@ -0,0 +1,335 @@
+---
+slug: /guides/custom-post-types
+title: Setting up Custom Post Types (CPTs) in Faust
+description: How to use Faust Templates to display your Custom Post Types
+---
+
+This guide covers how to set up Custom Post Types in your Faust templates.
```

Comment:
Are we using "Faust" or "Faust.js" in docs like these? (this is more for my own knowledge).

Reply:
We have been using "Faust" more often (Faust templates, Faust filters, etc).